benchmarks/benchmark_serving.py

```diff
@@ -100,6 +105,47 @@ def sample_requests(
     return sampled_requests
 
 
+def request_from_file(
+    dataset_path: str,
+    num_requests: int,
+    tokenizer: PreTrainedTokenizerBase,
+) -> List[Tuple[str, int, int]]:
+
+    dataset_args = TextDatasetArgs(num_samples=num_requests,
+                                   fixed_output_len=256)
+
+    # Load data.
+    with open(dataset_path) as f:
+        dataset_text = " ".join(f.readlines())
+    dataset_text = dataset_text[:dataset_args.max_len]
+
+    # format into chat convo
+    system_message = {
+        "content": "You are a chatbot with the explicit goal of "
+        "helping the user as best as possible",
+        "role": "system",
+    }
```

Comment:
FYI this actually won't work with Mixtral since it doesn't have a "system" role.